hphp/test/slow/include/fast-stat-cache.php.expect

```diff
@@ -0,0 +1 @@
+OldOldNew
```

Comment:
@wwbmmm 'DelayedStatCache' is fine.  Can you add some documentation about the option in hphp/doc/options.compiled?  Especially about the semantic change and when the option is helpful.

I think if you update the source code without restarting HHVM, having this cache may help consistency.  If the file updates all happen within the cache window, HHVM will either see the set of old files, or the set of new files, but never a mixture of both.